pypit/arproc.py

```diff
@@ -504,10 +508,12 @@ def flatnorm(slf, det, msflat, maskval=-999999.9, overpix=6, plotdesc=""):
     from pypit import arcyutils
     from pypit import arcyextract
     from pypit import arcyproc
+    dnum = 'det{0:02d}'.format(det)
```

Comment:
Am seeing this line of code here and there.  Am wondering if it should
be a method in case our convention changes someday.